code/src/containers/language-select/LanguageSelect.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { act, fireEvent, screen } from "@testing-library/react";
+
+import LanguageSelect from "@/containers/language-select/LanguageSelect";
+
+import locales from "@/constants/locales";
+import renderWithProviders from "@/utils/render-with-providers/renderWithProviders";
+
+describe("LanguageSelect", () => {
+  let select: HTMLElement;
```

Comment:
you define `let` only here and reasign it once. Maybe to create full realization like `let select = screen.getByRole("combobox")` ?

Reply:
Actually, we cannot because we need to use it after rendering the component